src/components/Chip/Chip.stories.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import Chip from 'components/Chip/Chip';
+
+storiesOf('Chip', module)
+  .add('Calcium', () => (
+    <Chip name="calcium" />
+  ))
+  .add('Vitamin B2', () => (
```

Comment:
Unnecessary stories here, one should be enough. 

Any time we're just showing different text for a component, we should reconsider the use of storybook. It a good indication that we're using storybook incorrectly. It's meant to show different states and not different text.

Reply:
Talked offline and agreed with the point made.